test/integration/epacems_test.py

```diff
@@ -0,0 +1,65 @@
+"""tests for pudl/output/epacems.py loading functions."""
+from pathlib import Path
+
+import dask.dataframe as dd
+import pytest
+
+from pudl.convert.epacems_to_parquet import epacems_to_parquet
+from pudl.etl import get_flattened_etl_parameters
+from pudl.output.epacems import epacems
+
+
+@pytest.fixture(scope='module')
+def cems_year_and_state(etl_params):
+    """Find the year and state defined in pudl/package_data/settings/etl_fast.yml."""
+    # the etl_params data structure alternates dicts and lists so indexing is a pain.
+    eia_epa = [item['datasets']
+               for item in etl_params['datapkg_bundle_settings'] if 'epacems' in item['name']]
+    cems = [item for item in eia_epa[0] if 'epacems' in item.keys()]
+    cems = cems[0]['epacems']
+    return {'years': cems['epacems_years'], 'states': cems['epacems_states']}
+
+
+@pytest.fixture(scope='session')
+def epacems_parquet_path(
+    datapkg_bundle,
+    pudl_settings_fixture,
+    pudl_etl_params,
+    request,
+    live_dbs,
+):
+    """Get CEMS path and convert a small amount of EPA CEMS data to parquet format."""
+    epacems_datapkg_json = Path(
+        pudl_settings_fixture['datapkg_dir'],
+        pudl_etl_params['datapkg_bundle_name'],
+        'epacems-eia',
+        "datapackage.json"
+    )
+    flat = get_flattened_etl_parameters(
+        pudl_etl_params["datapkg_bundle_settings"]
+    )
+    out_dir = Path(pudl_settings_fixture['parquet_dir'], 'epacems')
+    if not live_dbs:
+        # The test .parquet files are created by etl_test.py.
+        # But because pytest runs the tests in alphabetical order, this module runs before etl_test.py
+        # So I have to create the .parquet files here in order to run tests on them.
+        # Surely there is a better way; this doesn't belong here.
+        epacems_to_parquet(
+            datapkg_path=epacems_datapkg_json,
+            epacems_years=flat["epacems_years"],
+            epacems_states=flat["epacems_states"],
+            out_dir=out_dir,
+            compression='snappy',
+            clobber=False,
+        )
+    return out_dir
+
+
+def test_epacems_subset(cems_year_and_state, epacems_parquet_path):
+    """Minimal integration test of epacems(). Check if it returns a DataFrame."""
+    path = epacems_parquet_path
+    # initially I checked len() exactly, but that had to be hardcoded for a specific year/state.
+    # This is less strict, but because etl_fast only tests a single year/state, I think just as effective.
+    actual = epacems(columns=["gross_load_mw"], cems_path=path, **cems_year_and_state)
+    assert isinstance(actual, dd.DataFrame)
+    assert len(actual.compute()) > 0
```

Comment:
We have pytest set up to allow the user to point it at any settings file, and that determines what happens in the ETL within the tests. And we have another set of testing parameters besides `etl_fast.yml` in `etl_full.yml` which are also used for more exhaustive testing before merging into `main` or doing a release, and those parameters include all the years and all the states, so you probably want to make sure that whatever you have in the tests behaves reasonably on a single state-year, or on all of the data at once.

So maybe just pull the years out of the `etl_params` and hard-code `states=["ID"]` since it'll always be around in both?  We just use Idaho since it has the least data of any state.